assets/javascripts/discourse/initializers/discourse-algolia.js

```diff
@@ -241,64 +286,35 @@ export default {
             this.siteSettings.algolia_enabled &&
             this.siteSettings.algolia_autocomplete_enabled
           ) {
-            schedule("afterRender", () => {
-              document.body.classList.add("algolia-enabled");
-
-              const searchBox = document.querySelector("#search-box");
-              searchBox &&
-                searchBox.addEventListener(
-                  "focus",
-                  this._selectSearchBoxContent
+            return loadScript(
+              "/plugins/discourse-algolia/javascripts/autocomplete.js"
+            )
+              .then(() => {
+                return loadScript(
+                  "/plugins/discourse-algolia/javascripts/algoliasearch.js"
```

Comment:
Can you see if we can wrap this up in a `Promise.all` so that we can load the scripts in parallel?